appeals/web/src/server/appeals/personal-list/personal-list.mapper.js

```diff
@@ -317,6 +328,23 @@ export function mapAppealStatusToActionRequiredHtml(
 
 			return [lpaStatementAction, ipCommentsAction].join('<br>');
 		}
+		case APPEAL_CASE_STATUS.FINAL_COMMENTS: {
+			const lpaReceived = lpaFinalCommentsStatus === 'received';
+			const appellantReceived = appellantFinalCommentsStatus === 'received';
+
+			if (!lpaReceived && !appellantReceived) {
+				return 'Awaiting final comments';
+			}
+
+			const lpaAction = lpaReceived
+				? '<a class="govuk-link" href="#">Review LPA final comments</a>'
```

Comment:
links tbc?

Reply:
Yeah the journey doesn't exist yet and I think the routes aren't confirmed. We can just update this while building the review journey.